internal/server/iigointernal/base_president_test.go

```diff
@@ -9,42 +9,85 @@ import (
 )
 
 type allocInput struct {
-	resourceRequests   map[int]int
-	commonPoolResource int
+	resourceRequests   map[shared.ClientID]shared.Resources
+	commonPoolResource shared.Resources
 }
 
 func TestAllocationRequests(t *testing.T) {
 	cases := []struct {
 		name  string
 		input allocInput
-		reply map[int]int
+		reply map[shared.ClientID]shared.Resources
 		want  error
 	}{
+		// FIXME: @mpardalos could not get this running
```

Comment:
@Neelesh99 @preetl fyi

Reply:
Had a look, it seems the tests were written with an int version of resources in mind. So the expected results which we put in are effectively a floor function on the actual float numbers. Will fix and request to push

Reply:
Ok tests are fixed ;)